### ISSUE TITLE
tracking urls as part of page views

### ISSUE THREAD

**Issue by nareshv** [AmazingDreams/vue-matomo#60]
https://github.com/AmazingDreams/vue-matomo/blob/7dcd4fdfc1ba9b7f3465bdf803a218a53543c381/src/index.js#L37

Above code only takes care of the meta title (if defined in the router), Is it possible to use this code at https://developer.matomo.org/guides/spa-tracking to add page tracking with urls as well ?

```
        _paq.push(['setCustomUrl', '/' + window.location.hash.substr(1)]);
        _paq.push(['setDocumentTitle', 'My New Title']);
        _paq.push(['trackPageView']);
```

**Comment by AmazingDreams** [comment]
You mean you wish to set custom urls in the route meta?

Naresh Kumar Vudutha <notifications@github.com> schreef op 16 april 2020 17:10:53 CEST:

**Comment by nareshv** [comment]
I am wondering if thats possible.

Events vs Pageviews in SPA applications

**Comment by AmazingDreams** [comment]
This issue should have been fixed in 3.13.5-0
